.github/workflows/lint.yml

```diff
@@ -34,32 +34,32 @@
 #           # list of changed files within `super-linter`
 #           fetch-depth: 0
 
-#       ################################
-#       # Install packages #
-#       ################################
-#       - name: Install packages
-#         run: npm ci
-#       ################################
-#       # Lint codebase #
-#       ################################
-#       - name: Run ESLint
-#         run: npx lint-staged
-#       ################################
-#       # Check Prettier on codebase #
-#       ################################
-#       - name: Run Prettier
-#         run: npx prettier --check .
-#       ################################
-#       # Check for TypeScript errors #
-#       # TODO: Add this back once outstanding issues are resolved by all devs.
-#       ################################
-#     #   - name: Run TypeScript compiler (tsc) on staged files
-#     #     run: |
-#     #       # Get list of staged TypeScript files
-#     #       files=$(git diff --cached --name-only --diff-filter=d | grep '\.tsx\?$')
+      ################################
```

Comment:
is this accurately needed in this pr? this seems like some linting changes that should be in a different pr, but if it's necessary for this pr, thats fine.